tests/test_Guards.py

```diff
@@ -329,6 +329,18 @@ def test_Guards__safer_getattr__5():
     ) == str(err.value)
 
 
+def test_Guards__safer_getattr_raise():
+    from types import SimpleNamespace
+
+    from RestrictedPython.Guards import safer_getattr_raise as ga
+
+    o = SimpleNamespace(a="a")
+    assert ga(o, "a") == "a"
+    assert ga(o, "b", None) is None
+    with pytest.raises(AttributeError):
+        ga(o, "b")
```

Comment:
```suggestion
    from RestrictedPython.Guards import safer_getattr_raise

    o = SimpleNamespace(a="a")
    assert safer_getattr_raise(o, "a") == "a"
    assert safer_getattr_raise(o, "b", None) is None
    with pytest.raises(AttributeError):
        safer_getattr_raise(o, "b")
```

Using an alias makes the test unneccessary hard to read.